changelog/trait_getUniqueIdentifier.dd

```diff
@@ -0,0 +1,10 @@
+Add a compiler trait allowing to get an unique identifier
+
+A new `getUniqueIdentifer` trait allows to obtain an unique identifier,
+suitable to name a variable or a custom type.
+It takes a single parameter that must, after CTFE, represent a string used a preffix.
+
```

Comment:
Perhaps worth mentioning here and in the traits docs http://clhs.lisp.se/Body/f_gensym.htm.